### PR TITLE
CLDR-14400 Vue.js aboutbox, debug webpack

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14400
- run prettier on webpack config
- set webpack to debug mode (for now!)
- add a Vue-based about to the ST about page
- change cldrGui to not wipe out the entire <body>

### CODE REVIEW

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ private void serveWaitingPage(HttpServletRequest request, PrintWriter out, Surve
          * the front end could be in charge of the JavaScript.
          */
         String url = request.getContextPath() + request.getServletPath();
-        out.write("<html lang='" + SurveyMain.TRANS_HINT_LOCALE.toLanguageTag() + " class='claro'>\n");
+        out.write("<html lang='" + SurveyMain.TRANS_HINT_LOCALE.toLanguageTag() + "' class='claro'>\n");
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:140]: @btangmu this was an old bug, causes an invalid HTML page during loading

**Comment by srl295** [comment]
no real substance in the aboutbox yet. again, an incremental step, of getting the loader in place.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:140]: good catch!

File: tools/cldr-apps/js/src/views/About.vue [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<template>
+    <div id="home">
```
Comment (btangmu) [inline tools/cldr-apps/js/src/views/About.vue:2]: one small file, one giant leap for Survey Tool

File: tools/cldr-apps/js/webpack.config.js [hunk 3]
```diff
@@ -1,17 +1,28 @@
-const path = require('path');
+const path = require("path");
+const { VueLoaderPlugin } = require("vue-loader");
```
Comment (btangmu) [inline tools/cldr-apps/js/webpack.config.js:2]: just curious, would `import` instead of `require` work here?

**Review by btangmu — APPROVED** [review]

File: tools/cldr-apps/js/webpack.config.js (hunk 3 above)
Comment (srl295) [reply tools/cldr-apps/js/webpack.config.js:2]: @btangmu not sure

**Pull request merged**